genestack_client/files_util.py

```diff
@@ -1,27 +1,28 @@
 # -*- coding: utf-8 -*-
 
-from genestack_client import (GenestackException, Metainfo, Application, SudoUtils, FileFilter, validate_constant,
-                              GenomeQuery)
+from genestack_client import (GenestackException, Metainfo, Application, SudoUtils, FileFilter, validate_constant)
 
 CALCULATE_CHECKSUMS_KEY = 'genestack.checksum:markedForTests'
 EXPECTED_CHECKSUM_PREFIX = 'genestack.checksum.expected:'
 FILE_BATCH_SIZE = 500
 
 
-class SpecialFolders:
+class SpecialFolders(object):
     """
-    - ``IMPORTED``: folder where new files are created by Data Importers
-    - ``CREATED``: default folder for files created by ``Preprocess`` and ``Analyse`` applications
-    - ``TEMPORARY``: folder for temporary files
-    - ``UPLOADED``:  folder where raw files are stored
+    - ``IMPORTED``: folder with files created by Data Importers
+    - ``CREATED``: folder with files created by ``Preprocess`` and ``Analyse`` applications
+    - ``TEMPORARY``: folder with temporary files
+    - ``UPLOADED``: folder with uploaded raw files
+    - ``MY_DATASETS``: folder with created datasets
     """
     IMPORTED = 'imported'
     CREATED = 'created'
     TEMPORARY = 'temporary'
     UPLOADED = 'uploaded'
+    MY_DATASETS = 'my datasets'
 
 
-class SortOrder:
+class SortOrder(object):
```

Comment:
How this code related to commit message "files_util: Create datasets in 'My datasets' folder by default"?